src/utils.js

```diff
@@ -0,0 +1,14 @@
+export const sortByField = (array, fieldName, isReversed = false) => {
+    const copy = [...array]
+    return copy.sort((a, b) => {
+        const fieldA = a[fieldName];
+        const fieldB = b[fieldName];
+        
+        if (fieldA > fieldB) {
+            return isReversed ? -1 : 1;
+        } else if (fieldA < fieldB) {
+            return isReversed ? 1 : -1;
+        }
+        return 0;
```

Comment:
Nitpick (personal preference):
I'd recommend refactoring these if/else statements to make this logic slightly easier to read:
```js
  const fieldA = a[fieldName];
  const fieldB = b[fieldName];

  if (fieldA === fieldB) return 0;

  if (fieldA > fieldB) return isReversed ? -1 : 1;
  
  return isReversed ? 1 : -1;
}
```